src/app.js

```diff
@@ -45,29 +47,31 @@ app.get("/events", (req, res, next) => {
     "12M": 12,
   };
 
-  let { date_range, event_name } = req.query;
+  let { dateRange, eventName, aggregationType } = req.query;
   try {
     let result = redshift.getAggregatedEventsBy(
-      TIMEUNIT_BY_RANGE[date_range],
-      "total", // TODO implement other Aggregation Types
+      TIMEUNIT_BY_RANGE[dateRange],
+      aggregationType,
       {
-        previous: PREVIOUS_BY_RANGE[date_range],
-        event_name,
+        previous: PREVIOUS_BY_RANGE[dateRange],
+        eventName,
       },
     );
 
+    res.set("Access-Control-Allow-Origin", "*");
```

Comment:
What is going on here? Is there some kind of CORS issue you are anticipating?

Reply:
I got CORS issues when I was testing it. I guess we won't be needing this when we run everything in S3